qiskit_metal/qlibrary/couplers/cap_n_interdigital_tee.py

```diff
@@ -80,6 +80,10 @@ class CapNInterdigitalTee(QComponent):
                            layer='1')
     """Default connector options"""
 
+    TOOLTIP = """Generates a three pin (+) structure comprised of a primary two pin CPW
```

Comment:
Some TOOLTIPS have just one double quote, others have triple double quote.

TOOLTIP = " *** " 
vs
TOOLTIP = """ *** """

Note sure of the consequence of either.  However, I think it would be good to be consistent.